### PR TITLE
make it backword compatible by using utilities from ViewCompat

### PR DESCRIPTION
**Pull request by adilo**

### CODE REVIEW

File: library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt [hunk 1]
```diff
@@ -97,15 +99,21 @@ internal class CalendarAdapter(
...
                 calView.monthPaddingStart, calView.monthPaddingTop,
                 calView.monthPaddingEnd, calView.monthPaddingBottom
             )
             root.layoutParams = ViewGroup.MarginLayoutParams(LP.WRAP_CONTENT, LP.WRAP_CONTENT).apply {
                 bottomMargin = calView.monthMarginBottom
                 topMargin = calView.monthMarginTop
-                marginStart = calView.monthMarginStart
-                marginEnd = calView.monthMarginEnd
+
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
```
Comment (kizitonwose) [inline library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt]: This check is no longer needed since it'll always be `true`.

Comment (adilo) [reply library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt]: Sure thing. However, it is necessary in order to work properly on api 15 with library override mechanism. So I suggest you keep it. What do you think ?

Comment (kizitonwose) [reply library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt]: You know what? Let's proceed with the min SDK change, guess you really want it. I'll close this PR and just reopen the other one.

Comment (adilo) [reply library/src/main/java/com/kizitonwose/calendarview/ui/CalendarAdapter.kt]: Thanks!

**Pull request closed without merging**